sass/style.css

```diff
@@ -358,19 +419,33 @@ h3.city {
 
 .navbar.ftm {
   background: #4169E1;
+  padding: 0 .75rem;
 }
 
 .navbar.ftm .nav-link,
 .navbar.ftm .btn {
   font-size: 0.9rem;
 }
 
+.navbar.ftm .navbar-brand,
+.navbar.ftm .add-donation-site-form {
+  display: flex;
+  min-height: 100%;
+  align-items: center;
+}
+
 .navbar.ftm .navbar-brand {
   flex-direction: row;
+  align-items: center;
```

Comment:
You already added align-items in the previous CSS block.

Reply:
fixed redundancy!